sshd-core/src/main/java/org/apache/sshd/client/channel/PtyCapableChannelSession.java

```diff
@@ -239,6 +255,21 @@ protected void doOpenPty() throws IOException {
             writePacket(buffer);
         }
 
+        if (xForwarding) {
```

Comment:
Check that the X11 host & port properties are set. If not, it's futile to request X11 forwarding.

Also, I think there will need to be a way to pass in the cookie (from the .Xauthority file).